ngx-fudis/projects/dev/src/app/components/formExamples.component.ts

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable no-underscore-dangle */
+import { Component, DestroyRef, OnInit, inject } from '@angular/core';
+
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { TranslocoService } from '@ngneat/transloco';
+import { FudisDropdownOption, FudisRadioButtonOption } from 'dist/ngx-fudis/lib/types/forms';
+import { FudisErrorSummaryService, FudisTranslationConfigService } from 'ngx-fudis';
+import { Subject, takeUntil } from 'rxjs';
+
+type MyForm = {
+	dropdown: FormControl<FudisDropdownOption | null>;
+	textInput: FormControl<string | null>;
+	truth: FormControl<boolean | null>;
+	date: FormControl<Date | null>;
+	autocompleteDropdown: FormControl<FudisDropdownOption | null>;
+	autocompleteSearch: FormControl<FudisDropdownOption | null>;
+};
+
+@Component({
+	selector: 'app-form-examples',
+	templateUrl: 'formExamples.component.html',
+})
+export class AppFormExampleComponent implements OnInit {
```

Comment:
Pitäiskö tää AppFormExampleComponent olla otettu käyttöön tuolla `app.component.ts` vai missä tän pitäis näkyä?